### PR TITLE
Khoa work

### PR DESCRIPTION
**Pull request by Khoawala**
Temporarily putting the sort bar above the result list and bringing back the clicking functionality on the left side of the map.

### CODE REVIEW

**Comment by galiat** [comment]
image

fixes the issue that @mzxbliss noticed

**Comment by Khoawala** [comment]
> image
> 
> removing `left: 30%` fixes the issue that @mzxbliss noticed

Ah dammit... I just did this but am also working on fixing the header lol.

**Comment by Khoawala** [comment]
I made the changes galia mentioned

File: src/components/SplitScreen.module.css [hunk 1]
```diff
@@ -46,10 +46,11 @@
     }
     .slidingPane {
       background-color: rgba(255, 255, 255, 0.9);
-      bottom: -70%;
```
Comment (galiat) [inline src/components/SplitScreen.module.css]: curious about this change:

I might be causing this problem

**Comment by Khoawala** [comment]
Updated pull request

**Pull request merged**